scripts/commands/setmod.lua

```diff
@@ -0,0 +1,45 @@
+---------------------------------------------------------------------------------------------------
+-- func: setmod
+-- desc: Sets the specified modifier to teh specified value on the cursor target or player
+---------------------------------------------------------------------------------------------------
+
+require("scripts/globals/status")
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "sis"
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!setmod {modID} {optional target}")
```

Comment:
!setmod {modID} **{mod_amount}** {optional target}

Missing a parameter in the error readout, it might be confusing to peeps that use it in game to see what the inputs should be.

Reply:
nice catch fixing message

Reply:
fixed also fixed my "the" in the header and made the variable naming a little easier on the eyes.